sdk/src/main/java/com/ifountain/opsgenie/client/model/schedule/UpdateScheduleRequest.java

```diff
@@ -1,43 +1,51 @@
 package com.ifountain.opsgenie.client.model.schedule;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.ifountain.opsgenie.client.OpsGenieClientConstants;
 import com.ifountain.opsgenie.client.OpsGenieClientValidationException;
 import com.ifountain.opsgenie.client.model.BaseRequest;
 import com.ifountain.opsgenie.client.model.ObjectWithTimeZone;
+import com.ifountain.opsgenie.client.model.beans.DataWithName;
+import com.ifountain.opsgenie.client.model.beans.IdentifierType;
 import com.ifountain.opsgenie.client.model.beans.ScheduleRotation;
-
-
 import java.util.List;
+import java.util.Map;
+import java.util.Objects;
 import java.util.TimeZone;
+import java.util.HashMap;
 
 /**
  * Container for the parameters to make an update schedule api call.
  *
  * @see com.ifountain.opsgenie.client.IScheduleOpsGenieClient#updateSchedule(UpdateScheduleRequest)
  */
 public class UpdateScheduleRequest extends BaseRequest<AddScheduleResponse, AddScheduleRequest> implements ObjectWithTimeZone {
-    private String id;
+    @JsonIgnore
+    private String identifier;
```

Comment:
Any reason we have added `@JsonIgnore` for this two variables

Reply:
Because as per API request mentioned in docs, this should be added as a parameter and not added in body. We are creating body like this : apiConfiguration.setJson(JsonUtils.toJson(request));
So that's why @JsonIgnore is added here.